tests/unit/VueTypeaheadBootstrapList.spec.js

```diff
@@ -160,6 +165,14 @@ describe('VueBootstrapTypeaheadList', () => {
     expect(wrapper.findComponent(VueTypeaheadBootstrapListItem).vm.htmlText).toBe(`<span class='vbt-matched-text'>Cana</span>da`)
   })
 
+  it('Highlights text matches when query text contains regex escape characters', async () => {
```

Comment:
@mattzollinhofer ok, here is the test case

```
FAIL tests/unit/VueTypeaheadBootstrapList.spec.js
  ● Console

    console.error node_modules/vue/dist/vue.runtime.common.js:589
      [Vue warn]: Error in config.errorHandler: "SyntaxError: Invalid regular expression: /[CⒸＣĆĈĊČÇḈƇȻꜾ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ][nⓝｎǹńñṅňṇņṋṉƞɲŉꞑꞥ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ][dⓓｄḋďḍḑḓḏđƌɖɗꝺ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ] ([CⒸＣĆĈĊČÇḈƇȻꜾ]/: Unterminated group"
    console.error node_modules/vue/dist/vue.runtime.common.js:1739
      SyntaxError: Invalid regular expression: /[CⒸＣĆĈĊČÇḈƇȻꜾ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ][nⓝｎǹńñṅňṇņṋṉƞɲŉꞑꞥ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ][dⓓｄḋďḍḑḓḏđƌɖɗꝺ][aⓐａẚàáâầấẫẩãāăằắẵẳȧǡäǟảåǻǎȁȃạậặḁąⱥɐ] ([CⒸＣĆĈĊČÇḈƇȻꜾ]/: Unterminated group
          at new RegExp (<anonymous>)
          at VueComponent.highlightQuery (/home/runner/work/vue-typeahead-bootstrap/vue-typeahead-bootstrap/src/components/VueTypeaheadBootstrapList.vue:231:1)
```

Reply:
not sure where you want to go from here, but i made a separate PR that does two things:
1) reverts https://github.com/mattzollinhofer/vue-typeahead-bootstrap/pull/100
2) adds the same test case i added here, except in the other PR the test is correctly passing (where here it fails)

https://github.com/mattzollinhofer/vue-typeahead-bootstrap/pull/128

Reply:
That's exactly what I was hoping for, thank you!

Hopefully, this will help advance the support for other characters in the future.

Reply:
> 2. adds the same test case i added here, except in the other PR the test is correctly passing (where here it fails)

Out of curiosity, why do the tests fail here but pass on the other PR?

Reply:
@mattzollinhofer the tests are failing here because all i did was add the unit test, i didnt revert the code from https://github.com/mattzollinhofer/vue-typeahead-bootstrap/pull/100 that was causing the bug.

In the other PR, I do both 
1) revert https://github.com/mattzollinhofer/vue-typeahead-bootstrap/pull/100
2) add the test cases

Proving that reverting the PR does make the unit test pass